### DISCUSSION TITLE
Serverless Functions - edge caching 404, documentation error?

### DISCUSSION THREAD

**Discussion by rayshan** [vercel/vercel#5127] [General] [answered]
Hi! [Docs](https://vercel.com/docs/serverless-functions/edge-caching) say:

> Responses with a HTTP status code of 200 to a GET or HEAD request are cached by the Vercel Edge Network for the requested path. Other status codes are never cached.

But requests resulting in 404 are showing a cache HIT:

<img width="911" alt="Screen Shot 2020-08-30 at 11 14 45 AM" src="https://user-images.githubusercontent.com/3760840/91666681-8df65c80-eab3-11ea-90de-2e28812338da.png">

Is this expected?

BTW caching 404 would be my preference. I have an endpoint where data not found is meaningful, and I need the client to know about it for a 30 day TTL.

**Comment by mcsdevv** [comment, marked as answer]
Hey @rayshan, this has changed and we will update the documentation shortly - my apologies it wasn't correct.

We now cache 404s if `Cache-Control` tells us to cache it.

**Reply by rayshan** [reply]
Perfect thanks Matthew.
